config/job_conf.xml

```diff
@@ -314,7 +314,7 @@
     <!-- ICL -->
     <tool id="batman-nmr" destination="batman-container"/>
     <tool id="power-analysis" destination="papy-container"/>
-	<tool id="bruker2batman" destination="bruker2batman-container"/>
+	<tool id="bruker2batman" destination="dynamic-k8s-medium" resources="all"/>
```

Comment:
Excellent @vagelishand! Now you need to add the adequate part to https://github.com/phnmnl/container-galaxy-k8s-runtime/blob/develop/config/phenomenal_tools2container.yaml as explained in the same documentation you have been looking at.